tests/functional/test_textx_language.py

```diff
@@ -108,6 +108,92 @@ def test_unreferenced_abstract_rule():
     assert mm['Third']._tx_type == RULE_COMMON
 
 
+def test_abstract_rule_with_multiple_match_rule_refs():
+    """
+    Test that a single alternative of abstract rule can reference multiple
+    match rules
+    """
+    grammar = """
+    Rule: STRING|Rule1|ID|'#' INT '--';
+    Rule1: a='a'; // common rule
+    Prefix: '#';
+    Sufix: '--';
+    """
+    meta = metamodel_from_str(grammar)
+    model = meta.model_from_str('# 23 --')
```

Comment:
What if you pass 'a' instead of '# 12 - -' here?

Is model then instance of Rule1?

Maybe add another model_from_str/assert here to illustrate this

Reply:
Exactly. Added that case to tests and added the docs for these abstract rules changes.